addons/io_hubs_addon/components/operators.py

```diff
@@ -649,23 +650,23 @@ def execute(self, context):
         old_img = self.hubs_component[self.target_property]
 
         # Load/Reload the first image and assign it to the target property, then load the rest of the images if they're not already loaded. This mimics Blender's default open files behavior.
-        primary_filepath = os.path.join(dirname, self.files[0].name)
         primary_img = bpy.data.images.load(
-            filepath=primary_filepath, check_existing=True)
+            filepath=self.filepath, check_existing=True)
```

Comment:
To match Blender's default behavior this should use primary_filepath instead of self.filepath.

Reply:
I think in the 2. commit, I changed all that. It should have also restored the behaviour of the 1. file being selected, s.a.

Reply:
The behavior is still changed, see my above comment.

Reply:
Sorry, I'm new to pull requests. This commit: Better solution for open image with relative path failing had the original behaviour restored.

Reply:
Please consider this request obsolete, I created too much chaos with the branches and opened a new request for this topic, which includes your change requests
https://github.com/MozillaReality/hubs-blender-exporter/pull/266